tikv-client/src/main/java/com/pingcap/tikv/types/MySQLType.java

```diff
@@ -64,14 +64,16 @@
     }
   }
 
-  private int typeCode;
-  private long defaultSize;
-  private int prefixSize;
+  private final int typeCode;
+  private final long defaultSize;
+  private final int prefixSize;
+  private final int defaultM;
```

Comment:
defaultM can not explains itself.

Reply:
M is short for MySQL's length(as described in its document). I can change its name or add comments. Any good ideas?

Reply:
comments is ok.